model/task/task.go

```diff
@@ -2645,7 +2659,7 @@ func (t *Task) Archive() error {
 					},
 				},
 			},
-			updateDisplayTasksAndTasksBson(),
+			updateDisplayTasksAndTasksExpression,
```

Comment:
Where is this defined? My grepping is letting me down here.

Reply:
The file wasn't expanded and it of course wasn't in the code base yet. Found it!